cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -30,6 +30,491 @@ Cask Data Application Platform Release Notes
    :backlinks: none
    :depth: 2
 
+`Release 4.1.0 <http://docs.cask.co/cdap/4.1.0/index.html>`__
+=============================================================
+
+New Features
+------------
+
+- :cask-issue:`CDAP-4556` - CDAP now uses environment variables in the ``spark-env.sh`` and
```

Comment:
@awholegunch what's the ordering of these features? Can we pull the top features of a release to the top? I can mark them in this PR. Also, this particular one seems like an improvement.